packages/i18n/src/locales/en.i18n.json

```diff
@@ -3269,7 +3269,7 @@
   "Log_Trace_Subscriptions_Filter": "Trace subscription filter",
   "Log_Trace_Subscriptions_Filter_Description": "The text here will be evaluated as RegExp (`new RegExp('text')`). Keep it empty to show trace of every call.",
   "Log_View_Limit": "Log View Limit",
-  "Logged_Out_Banner_Text": "Your workspace admin ended your session on this device. Please log in again to continue.",
+  "Logged_Out_Banner_Text": "Your session was ended on this device, please log in again to continue.",
```

Comment:
We shouldn't change the content of this key, since it's being used in other places like the Device management page where the admin can log out users.

Reply:
This was an intentional change after discussion with design team, idea is that same force_logout banner is also used for different purposes, so we should make this text more generic.